assets/xml/scenes/overworld/spot00_pal_n64.xml

```diff
@@ -7,6 +7,15 @@
         <Cutscene Name="gHyruleFieldSouthEponaJumpCs" Offset="0xFA00"/>
         <Cutscene Name="gHyruleFieldWestEponaJumpCs" Offset="0x10570"/>
         <Cutscene Name="gHyruleFieldGateEponaJumpCs" Offset="0x10B50"/>
+        <Cutscene Name="gHyruleFieldZeldaFlashbackCs" Offset="0xA000"/>
+        <Cutscene Name="gHyruleFieldTitleScreenCs" Offset="0xA920"/>
+        <Cutscene Name="gHyruleFieldCreditsCs" Offset="0xBF44"/>
+        <Cutscene Name="gHyruleFieldBeforeCreditsCs" Offset="0xC8D4"/>
+        <Cutscene Name="gHyruleFieldGateEponaJump2Cs" Offset="0xC130"/>
```

Comment:
Are these both really the same cutscene? Or are they different cutscenes for the two different gates?

Reply:
the script is the same, maybe their tool did bulk exports and they forgot to remove this one?